### ISSUE TITLE
Pointer Speed adjustment of internal trackpad doesn't work on M2 (Pro) devices

### ISSUE THREAD

**Issue by Tiemo99** [linearmouse/linearmouse#492]
### OS

macOS 13.4 (22F66)

### LinearMouse

v0.9.1

### Describe the bug

Unfortunately it is not possible to change the pointer speed of the internal trackpad on a new MacBook Pro 16". (2023, M2 Pro) Manipulating the speed slider doesn't have any effect (independent of the acceleration setting which works as expected)

The issue only occurs on the new model with M2 chip whereas the speed setting works perfectly fine on my previous 14" MBP. (2021, M1 Pro) They are both equipped with the latest OS and LinearMouse version.

### Anything else?

For reference: I've been using LinearMouse for a long time on many different Macs and never had any issues. I always use the same speed and acceleration settings, so this shouldn't be a factor.

**Comment by lujjjh** [comment]
Thanks for your feedback!

Unfortunately, this is a known issue (#270, #279) and there is currently no workaround. If you find an application that can do this, please let me know.

**Comment by mpekarMPB** [comment]
Hi @lujjjh, I think I've figured out how to show the internal trackpad on the LinearMouse dashboard. I have new m2 macbook and the app works correctly. I've connected external mouse through bluetooth to swap the scrollwheel direction. App then shown Trackpad when I've disabled automatic switching. See screenshots:

![image](https://github.com/linearmouse/linearmouse/assets/98816633/cac1fc71-d074-44f9-bea0-895e2e8052d8)
![image](https://github.com/linearmouse/linearmouse/assets/98816633/0b02c8cf-4a64-4cee-87e1-dbb617864ded)
